tests/cases/fourslash/refactorConvertToGetAccessAndSetAccess17.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path='fourslash.ts' />
+
+//// class A {
+////     public _a: number = 1;
+////     /*a*/public a: string = "foo";/*b*/
+////     public b: number = 2;
+////     /*c*/public _b: string = "foo";/*d*/
+//// }
+
+goTo.select("a", "b");
+verify.not.refactorAvailable("Generate 'get' and 'set' accessors");
```

Comment:
i do not see why we would not allow this. you just need to come up with a temporary name that starts with `_a`, set the rename location to it, and let the user decide what they want to call it. you can just call `createUniqueName` to create you an identifier with a unique name and use it instead.

See https://github.com/Microsoft/TypeScript/blob/1ed30c6d4d78aeb7de0c81604dbf38120d82e82b/src/services/refactors/extractSymbol.ts#L1137-L1161